packages/fxa-content-server/app/scripts/lib/auth-errors.js

```diff
@@ -616,9 +616,7 @@ var ERRORS = {
   },
   ECOSYSTEM_ANON_ID_UPDATE_CONFLICT: {
     errno: 190,
-    message: t(
-      'Could not update Ecosystem Anon ID because precondition failed.'
-    ),
+    message: 'Could not update Ecosystem Anon ID because precondition failed.', // only used for logging, no need for translation
```

Comment:
Thanks for the comment, it will help future us :)

Reply:
This is what I get for copy-pasting!

Reply:
lol, don't feel bad - I missed it in the review :beers: